Common/TableProducer/tracksextraConverter.cxx

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+
+using namespace o2;
+using namespace o2::framework;
+
+// Converts TracksExtra table from version 000 to 001
+struct tracksextraConverter {
+  Produces<aod::StoredTracksExtra_001> tracksExtra_001;
+  void process(aod::TracksExtra_000 const& tracksExtra_000)
+  {
+
+    // dummy itsClusterSizes, fill with overflows if a hit in the layer is present
```

Comment:
Is overflow really the best choice? Not just `1` maybe?

Reply:
I think it would be more difficult to distinguish whether this is a dummy entry or not as it is much more probable to have  1-size clusters than > 14-size clusters.